src/NCronJob/Registry/JobDefinition.cs

```diff
@@ -48,13 +48,15 @@ internal sealed record JobDefinition(
     public RetryPolicyBaseAttribute? RetryPolicy => JobPolicyMetadata.RetryPolicy;
     public SupportsConcurrencyAttribute? ConcurrencyPolicy => JobPolicyMetadata.ConcurrencyPolicy;
 
-    private bool IsDisabled => CronExpression == NotReacheableCronDefinition ;
+    public bool IsDynamic => Type == typeof(DynamicJobFactory);
```

Comment:
Do we want to call that more user-centric?
 * `IsAnonymousJob`
 * `IsMinimalJob`
 * `IsLambdaDefinition`

Reply:
Let's go with `IsAnonymousJob`

Reply:
Done

Reply:
Giving it a bit more thinking, I believe `IsAnonymousJob` may be ill fitted. Anonymous literally means with "with no name" and from the standpoint of a recurring schedule where we surface both this property and the name, this doesn't play that well... :-/

In my opinion, `IsMinimalApi` is to tied to a asp.net use case

So, ... `IsAnonymousFunction`, `IsLambdaExpression`, `IsDelegate`, ...  ?

@linkdotnet Thoughts?

Reply:
Well we call it MinimalAPI as well. Not maybe best suited, but it follows exactly the same patterns.

Reply:
The closest might be `IsLambdaExpression` but that is very technical. `IsDelegate` is my favourite from the ones you suggested. Maybe `IsMinimalDefinition`/ `IsMinimalJob` (removing the API thingy.

Reply:
I'm still not sold with "Minimal". Minimal API pitch, from the official doco is "Minimal APIs are a simplified approach for building fast HTTP APIs with ASP.NET Core"

A lambda expression is just another way to build a delegate.

Googling "what is a delegate in programming?" brings lots of things on topic (pointer to a function, ...). However, besides the fact it's pretty close it's indeed pretty technical.

Taking it from a different angle, how about reversing the boolean? Maybe `IsIJob`, 'IsTypedJob` ?

Reply:
IsTypedJob might be the best. 
the documentation should make it super clear what we mean with that.

Reply:
Landed in #234